pr_testing/run-pr-comparisons

```diff
@@ -592,18 +592,19 @@ sed -i "s/^COMPARISON${TEST_FLAVOR_STR};RUNNING,/COMPARISON${TEST_FLAVOR_STR};$B
 mv ${RESULTS_FILE} $WORKSPACE/testsResults/
 
 [ -e comparisonDetails.txt ] && mv comparisonDetails.txt upload/
+REPORT_FILE=$WORKSPACE/testsResults/$(get_result_file_name "comp" "${TEST_FLAVOR}")
+NANO_REPORT_FILE=$WORKSPACE/testsResults/$(get_result_file_name "compnano" "${TEST_FLAVOR}")
 ${CMS_BOT_DIR}/report-pull-request-results "COMPARISON_READY" --report-url ${PR_RESULT_URL} \
   -f $WORKSPACE/${ERRORS_FILE} --f2 $WORKSPACE/wf_non_consistent_das.txt \
-  --missing_map $WORKSPACE/results/JR-comparison/missing_map.txt  --report-file $WORKSPACE/testsResults/20-comparison-report.res
+  --missing_map $WORKSPACE/results/JR-comparison/missing_map.txt  --report-file $REPORT_FILE
 
 if [ "${TEST_FLAVOR}" != "" ] ; then
-  sed -i -e "s|## Comparison Summary|## ${UC_TEST_FLAVOR} Comparison Summary|" $WORKSPACE/testsResults/20-comparison-report.res
-  mv $WORKSPACE/testsResults/20-comparison-report.res $WORKSPACE/testsResults/20-${TEST_FLAVOR}-comparison-report.res
+  sed -i -e "s|## Comparison Summary|## ${UC_TEST_FLAVOR} Comparison Summary|" $REPORT_FILE
```

Comment:
@iarspider , I think we need the `mv` statement ... right?
```
mv $REPORT_FILE $NANO_REPORT_FILE
```

Reply:
I don't think so - `NANO_REPORT_FILE` is `21-xxxx`, while `REPORT_FILE` is `20-xxxx`, they are different. The `mv` was removed on purpose, since `get_result_file_name` function takes care of naming `20-xxxx` file differently if `TEST_FLAVOR` is not empty